tests/testthat/test-get_source_expressions.R

```diff
@@ -0,0 +1,42 @@
+context("get_source_expression")
+
+test_that("tab positions have been corrected", {
+  f <- tempfile()
+
+  writeLines("1\n\t", f)
+  expect_error(lintr:::get_source_expressions(f), NA, label="empty parsed_content line")
```

Comment:
We can remove the `lintr:::`, also it feels like it might be more readable if the redundant code was to a `compare_parsed_content` function

```r
compare_parsed_content <- function(content, code) {
  f <- tempfile()
  on.exit(unlink(f))
  writeLines(content, f)
  pc <- get_source_expressions(f)[["expressions"]][[1L]][["parsed_content"]]
  force(code)
}